controllers/version_update.go

```diff
@@ -372,18 +372,19 @@ func parseImageString(imageString string) (*Version, error) {
 		return &Version{}, nil
 	}
 	splits := strings.Split(imageString, ":")
-	if len(splits) != 2 {
-		return nil, fmt.Errorf("failed to parse image string %s, not in expected format of xxx:xxx", imageString)
+	if len(splits) > 3 {
```

Comment:
It is better to change to use `LastIndex` to look for the last `:` to extract the version